filters/exclusions.txt

```diff
@@ -75,6 +75,8 @@ $popup,~third-party
 !
 !### Easylist ###
 !
+! We have ##.ad-unit:not(.textads) in our filters
+/^##\.ad-unit$/
```

Comment:
Better like this https://github.com/AdguardTeam/FiltersRegistry/pull/945/files#diff-6472c0fcd53f81660278097de5b81a5a1cd70c38b8a5068d02039207a61d5726L39

```suggestion
/^(~.*)?##\.ad-unit$/
```

It will match `~example.com##.ad-unit`